feature-libs/order/components/order-history/order-history.component.spec.ts

```diff
@@ -223,6 +262,37 @@ describe('OrderHistoryComponent', () => {
     expect(elements.length).toEqual(2);
     expect(component.sortType).toEqual('byDate');
   });
+
+  it('should display PO Number & Cost Center', () => {
+    spyOn(featureConfigService, 'isLevel')
```

Comment:
Do we need this when our mock service already returns `true` for `isLevel()`?

Reply:
we want to return true only if 5.1
spyOn(featureConfigService, 'isLevel')
      .withArgs('5.1')
      .and.returnValue(true)
      .withArgs('!5.1')
      .and.returnValue(false);

Reply:
Could you please point out where these args are inputted? I am not aware of us using version flags for unit tests in the CI. It also appears that this test should fail in the instance of `< 5.1` if this were the case.

Reply:
thanks, you are right we don't need 'spyOn'.  tested both 5.1 and 4.2.